source/labyrinths/castleofthecelestialknights.js

```diff
@@ -266,7 +266,7 @@ module.exports = new LabyrinthTemplate("Castle of the Celestial Knights",
 	},
 	{
 		// Rooms
-		"Event": ["Door 1 or Door 2?", "Twin Pedestals", "Imp Contract Faire", "Gear Collector", "The Score Beggar", "Apple Pie Wishing Well", "Workshop", "Merchant", "Rest Site", "Library", "Treasure"],
+		"Event": ["Door 1 or Door 2?", "Twin Pedestals", "Imp Contract Faire", "Gear Collector", "The Score Beggar", "Apple Pie Wishing Well", "Workshop", "Merchant", "Rest Site", "Library", "Treasure", "Let Sleeping Dogs Lie"],
 		"Battle": ["Gaia Knightess and Elegant Stella Fight?", "Luna Militissa Fight", "Meteor Knight Fight", "Spacedust Cadet Fight"],
```

Comment:
Since the name of Gaia Knightess Fight got changed, configs here and in the Everything Bagel should also be updated.
```suggestion
		"Battle": ["Gaia Knightess Fight", "Luna Militissa Fight", "Meteor Knight Fight", "Spacedust Cadet Fight"],
```